site/src/components/Margins/Margins.tsx

```diff
@@ -26,14 +26,11 @@ export const Margins: FC<MarginsProps> = ({
 	return (
 		<div
 			{...divProps}
-			css={{
-				marginLeft: "auto",
-				marginRight: "auto",
-				maxWidth: maxWidth,
-				paddingLeft: sidePadding,
-				paddingRight: sidePadding,
-				width: "100%",
+			style={{
+				...(maxWidth ? { maxWidth: `${maxWidth}px` } : {}),
+				...(sidePadding ? { padding: `${sidePadding}px` } : {}),
```

Comment:
same with these, these spreads are a bit noisy visually

instead you could do something like

```tsx
maxWidth: maxWidth ? `${maxWidth}px` : undefined,
padding: sidePadding ? `${sidePadding}px` : undefined,
```